src/main/java/org/apache/commons/lang3/StringUtils.java

```diff
@@ -2878,27 +2866,30 @@ public static int indexOfAnyBut(final CharSequence cs, final char... searchChars
      * StringUtils.indexOfAnyBut("aba", "ab")        = -1
      * </pre>
      *
-     * @param seq  the CharSequence to check, may be null
+     * @param srcChars  the CharSequence to check, may be null
      * @param searchChars  the chars to search for, may be null
      * @return the index of any of the chars, -1 if no match or null input
      * @since 2.0
      * @since 3.0 Changed signature from indexOfAnyBut(String, String) to indexOfAnyBut(CharSequence, CharSequence)
      */
-    public static int indexOfAnyBut(final CharSequence seq, final CharSequence searchChars) {
-        if (isEmpty(seq) || isEmpty(searchChars)) {
+    public static int indexOfAnyBut(final CharSequence srcChars, final CharSequence searchChars) {
+        if (srcChars == null || searchChars == null || srcChars.length() == 0 || searchChars.length() == 0) {
```

Comment:
Not sure why you inlined these calls. Keep the PR to only the required changes to fix a bug.